dts/bindings/adc/microchip,xec-adc.yaml

```diff
@@ -17,5 +17,10 @@ properties:
     "#io-channel-cells":
       const: 1
 
+    clk_time:
```

Comment:
Please change this to use `clk-time`, with a dash, to follow the general devicetree convention for property names.

Reply:
Updated property name.